vms/models/stock_move.py

```diff
@@ -14,3 +14,6 @@ class StockMove(models.Model):
         'vms.order.line', string='VMS Order Line',
         readonly=True, related='vms_product_line_id.order_line_id',
         store=True,)
+    unit_id = fields.Many2one(
+        'vms.order', related="vms_order.unit_id",
```

Comment:
Whi the relation is to vms.order if the field is unit_id?.

The field must be:

unit_id = fields.Many2one('fleet.vehicle', related="vms_order_line_id.order_id.unit_id")